src/components/movie-details/movie-details.tsx

```diff
@@ -0,0 +1,39 @@
+import styles from './styles.module.scss'
+
+interface Props {
```

Comment:
To be consistent with the components from the previous task:

```suggestion
interface MovieDetailsProps {
```